src/test/regress/sql/cont_trigger.sql

```diff
@@ -1,4 +1,6 @@
-CREATE CONTINUOUS VIEW cont_tg_cv AS SELECT count(*) FROM cont_tg_stream;
+select pg_sleep(2);
```

Comment:
The trigger process isn't started until after that sleep. The sleep was there before, I just moved it to the start. A better solution might be to block on creating triggers until the trigger proc is up.